test/CommonSpaces/unit_common_spaces.jl

```diff
@@ -75,14 +78,17 @@ using Test
     @test grid isa Grids.SpectralElementGrid2D
     @test Grids.topology(grid).mesh isa Meshes.EquiangularCubedSphere
 
-    space = ColumnSpace(;
-        z_elem = 10,
-        z_min = 0,
-        z_max = 1,
-        staggering = Grids.CellCenter(),
-    )
-    grid = Spaces.grid(space)
-    @test grid isa Grids.FiniteDifferenceGrid
+    # Column spaces are not supported with MPI
+    if !(ClimaComms.context() isa ClimaComms.MPICommsContext)
```

Comment:
Does this just not work?

Reply:
I had this error:
```
  AssertionError: Columns can only be created on Singleton contextes.
```

But maybe it's because I am not caught up to main. Let me try rebasing

Reply:
Actually, I added that `@assert`. I'll verify again

Reply:
I added the assert because creating columns in MPI settings is not possible:
https://github.com/CliMA/ClimaCore.jl/blob/c19162a39c1c8d0b5465449fc55ec78ac3632bc2/src/Topologies/interval.jl#L46

Reply:
Here's the failing build: https://buildkite.com/clima/climacore-ci/builds/5255